File: uri_exercicios_em_Cpp/beginner/the_greatest.cpp
#include <iostream>
#include <cmath>
using namespace std;

int the_greatest(int a,int b){
    return (a+b+abs(a-b))/2;
}

int main(){

    int a,b,c;

    cin >> a >> b >> c;

    cout << the_greatest(the_greatest(a,b),c) << " eh o maior" << endl;

    return 0;
}

File: beecrowd_exercicios_em_Java/src/beginner/TheGreatest.java
package beginner;

import java.io.IOException;
import java.util.*;
import java.math.*;

public class TheGreatest {
    static Scanner sc = new Scanner(System.in);

    public static int OMaior(int a, int b){
        return (a + b + Math.abs(a-b))/2;
    }

    public static void main(String[] args) throws IOException {

        int a = sc.nextInt(),b = sc.nextInt(), c = sc.nextInt();

        System.out.printf("%d eh o maior\n",OMaior(OMaior(a,b),c));


    }

}